tgui/packages/tgui/interfaces/Airlock.js

```diff
@@ -1,8 +1,417 @@
+              mt={0.5}
+              color="bad"
+              icon="plug"
+              disabled={!!backupTimeLeft}
+              onClick={() => act("disruptBackup")}>
+              Disrupt
+            </Button>
+          )}>
+          {backupTimeLeft ? "Offline" : "Online"}
+          {" "}
+          {(!wires.backup_1 || !wires.backup_2)
+            && "[Wires cut!]"
+            || (backupTimeLeft > 0
+              && `[${backupTimeLeft}s]`)}
+        </LabeledList.Item>
+      </LabeledList>
+    </Section>
+  );
+};
+
+const AccessAndDoorControl = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    mainTimeLeft,
+    backupTimeLeft,
+    wires,
+    idScanner,
+    boltsAreUp,
+    opened,
+    welded,
+  } = data;
+
+  const isDisabled = (data.status === 2);
+
+  return (
+    <Section title="Access and Door Control"
+      pt={1}>
+      <LabeledList>
+        <LabeledList.Item
+          label="ID Scan"
+          color="bad"
+          buttons={(
+            <Button
+              py={0.5}
+              width={6.7}
+              align="center"
+              color={idScanner ? "good" : "bad"}
+              icon={idScanner ? "power-off" : "times"}
+              disabled={!wires.idScanner
+                || (mainTimeLeft && backupTimeLeft)}
+              onClick={() => act("idScanToggle")}>
+              {idScanner ? "Enabled" : "Disabled"}
+            </Button>
+          )}>
+          {!wires.idScanner && "[Wires cut!]"}
+        </LabeledList.Item>
+        <LabeledList.Item
+          label="Door Bolts"
+          color="bad"
+          buttons={(
+            <Button
+              py={0.5}
+              mt={0.5}
+              width={6.7}
+              align="center"
+              color={!boltsAreUp ? "bad" : "good"}
+              icon={!boltsAreUp ? "unlock" : "lock"}
+              disabled={!wires.bolts
+                || (mainTimeLeft && backupTimeLeft) || (isDisabled)}
+              onClick={() => act("boltToggle")}>
+              {!boltsAreUp ? "Lowered" : "Raised"}
+            </Button>
+          )}>
+          {!wires.bolts && "[Wires cut!]"}
+        </LabeledList.Item>
+        <LabeledList.Item
+          label="Door Control"
+          color="bad"
+          buttons={(
+            <Button
+              py={0.5}
+              width={6.7}
+              align="center"
+              mt={0.5}
+              color={opened ? "bad" : "good"}
+              icon={opened ? "sign-out-alt" : "sign-in-alt"}
+              disabled={(!boltsAreUp || welded)
+                || (mainTimeLeft && backupTimeLeft) || (isDisabled)}
+              onClick={() => act("openClose")}>
+              {opened ? "Open" : "Closed"}
+            </Button>
+          )}>
+          {!!(!boltsAreUp || welded) && (
+            <span>
+              [Door is {!boltsAreUp ? "bolted" : ""}
+              {(!boltsAreUp && welded) ? " and " : ""}
+              {welded ? "welded" : ""}!]
```

Comment:
Possible to do something like `[Door is {!boltsAreUp && 'bolted'}` etc. rather than ternaries here?